portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //response.getWriter().println("<h1>Hello world!</h1>");
```

Comment:
Better to just remove this than comment it out.  Generally don't want to leave comment lines.

Reply:
Sounds good! Let me go ahead and do that.